deno-bootstrap/index.ts

```diff
@@ -0,0 +1,57 @@
+const scriptType = Deno.args[0];
+const script = Deno.args[1];
+
+const importURL =
+  scriptType == "import"
+    ? script
+    : "data:text/tsx," + encodeURIComponent(script);
+
+const server = Deno.listen({
+  hostname: "0.0.0.0",
+  port: 0,
+});
+
+const addr = server.addr as Deno.NetAddr;
+
+console.log(`deno-listening-port ${addr.port.toString().padStart(5, " ")} `);
+
+// Now that we're listening, start executing user-provided code. We could
+// import while starting the server for a small performance improvement,
+// but it would complicate reading the port from the Deno logs.
+const handler = await import(importURL);
+if (!handler.default) {
+  throw new Error("No default export found in script.");
+}
+if (typeof handler.default !== "function") {
+  throw new Error("Default export is not a function.");
+}
```

Comment:
We'd still have a `guest.ts` or similar, right, because we have different rules for exports there - the first import is taken and it doesn't have to be default.

Reply:
Ok yes, I was going to ask you about this. I just wrote up something that worked, but wasn't sure what might be best here. I'll extend it to support the first export as well?